src/test/java/bio/terra/service/snapshotbuilder/utils/ConceptChildrenQueryBuilderTest.java

```diff
@@ -5,26 +5,57 @@
 import bio.terra.common.CloudPlatformWrapper;
 import bio.terra.common.category.Unit;
 import bio.terra.model.CloudPlatform;
+import bio.terra.model.SnapshotBuilderDomainOption;
 import org.hamcrest.Matchers;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.EnumSource;
 
 @Tag(Unit.TAG)
 class ConceptChildrenQueryBuilderTest {
+
+  private SnapshotBuilderDomainOption createDomainOption(
+      String name, int id, String occurrenceTable, String columnName) {
+    var option = new SnapshotBuilderDomainOption();
+    option.name(name).id(id).tableName(occurrenceTable).columnName(columnName);
```

Comment:
TOL: I am a little sad we have to do this, because we lose all the nice pretty named parameters, and have to remember which parameters go in which order. Maybe at some point we can figure out a way to do this that preserves the named params.

Reply:
> I am a little sad we have to do this, because we lose all the nice pretty named parameters, and have to remember which parameters go in which order

Do mean you'd rather use fluent style constructors over a positional style constructor? I don't see why we have to do it one way the other here, either could work. Positional style is more concise and, while I agree that some legibility is lost, I personally find it better for shorter argument lists. When working in an IDE it's easy to see which parameter lines up with which argument (using `Parameter Info` in intellij).

For me, part of the problem here is that we're using openapi generated models throughout our code. This means that we have to make do with the generated code's conventions, and we aren't insulating our logic from changes to the API model. I think we'd be better off converting the generated models to custom code at the outer (controller) layer which would give us more control over decisions like this. But that would be a bigger change and can wait until we have more of the API written.

Reply:
Yeah, I tend to prefer the fluent style constructors. Or avoiding this helper construction method. But I agree that maybe converting generated models to custom code at the controller level would be a good thing.